### PR TITLE
[IMP] subscriptions: update close subs doc for 17

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3676411&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by ksc-odoo** [comment]
@meng-odoo This one is ready for its first round of Peer Review - whenever you get a chance! 👍

**Review by meng-odoo — APPROVED** [review]
Hi @ksc-odoo! I just had small wording suggestions for this doc. Please go ahead and move it to the next review stage once you've gone through my suggestions. Thanks!

**Comment by ksc-odoo** [comment]
Thanks for the awesome feedback @meng-odoo -- I made all the suggested edits, and believe this one is now ready for a 2nd Round of Peer Review from the materials team. So, whenever any of y'all get a chance to check this one out, I'd really appreciate it! 👍 @Felicious @hojo-odoo @brse-odoo

File: content/applications/sales/subscriptions/closing.rst [hunk 1]
```diff
@@ -1,105 +1,118 @@
...
-  :alt: Configuration to close your subscription with Odoo Subscriptions
+  :alt: The Closable option on a recurring plan form in Odoo Subscriptions.
 
 .. note::
-   Be sure to check out our documentation on
-   how to :doc:`Use subscription templates <plans>`
-   to fully understand the importance of this feature in a basic flow using the
-   **Odoo Subscriptions** application.
+   Check out our documentation on :doc:`subscription plans <plans>` for more information on how to
+   configure recurring plans.
```
Comment (Felicious) [inline content/applications/sales/subscriptions/closing.rst:38]: Very minor thing, but I was once put on blast for linking to other docs this way. Apparently, we **must** link to other doc [this way](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#see-also)...

```suggestion
   .. seealso::
   :doc:`Configure recurring plans <plans>`
```

File: content/applications/sales/subscriptions/closing.rst [hunk 2]
```diff
@@ -1,105 +1,118 @@
...
+Close a subscription
+====================
 
 Administrator view
 ------------------
 
-Once confirmed, a quotation becomes a sales order and a new subscription is automatically created.
-Therefore, this subscription has the status *In progress*. From there, you have the possibility to
-close the subscription.
+After a quotation for a subscription product(s) has been confirmed, it becomes a sales order, and
```
Comment (Felicious) [inline content/applications/sales/subscriptions/closing.rst:46]:
```suggestion
After a quotation for a subscription product has been confirmed, it becomes a sales order, and
```

File: content/applications/sales/subscriptions/closing.rst [hunk 3]
```diff
@@ -1,105 +1,118 @@
...
-  :alt: Close your subscription from an administration point of view with Odoo Subscriptions
+  :alt: Close subscription from an administration point of view with Odoo Subscriptions.
 
-By using the smart button *Close*, you have to enter a close reason. For example, "Subscription too
-expensive", "Subscription does not meet my requirements", "Subscription reached its end date", etc.
-Immediately after confirming your close reason, you can observe that the status of the subscription
-is now *Closed* and that the close reason is mentioned on the subscription.
+When the :guilabel:`Close` button is clicked, a :guilabel:`Close Reason` pop-up window appears. In
+this pop-up window, enter the reason for closing the subscription or select an option from the
+drop-down menu of options that appears when the :guilabel:`Reason` field is clicked.
```
Comment (Felicious) [inline content/applications/sales/subscriptions/closing.rst:59]:
```suggestion
Clicking the :guilabel:`Close` button prompts a :guilabel:`Close Reason` pop-up window to appear, allowing administrators to input the reason for closing the subscription, or choose from the drop-down menu of options in the :guilabel:`Reason` field.
```

**Review by Felicious — COMMENTED** [review]
Hi @ksc-odoo ! I got through reviewing about half of this PR, I'll get to the rest tomorrow (:

Amazing job so far! I just had super minor rewording suggestions

File: content/applications/sales/subscriptions/closing.rst [hunk 4]
```diff
@@ -1,105 +1,118 @@
...
+Customer view
+-------------
+
+.. note::
+  As an administrator, the ability to visualize what customers see when managing their subscriptions
+  is accessible via the :guilabel:`Preview` button, located at the top of the subscription sales
+  order.
+
+From the customer's point of view, in the customer portal, the :guilabel:`Close Subscription` button
+is located on the left side of the sales order.
```
Comment (Felicious) [inline content/applications/sales/subscriptions/closing.rst:90]:
```suggestion
  Administrators can use the :guilabel:`Preview` button at the top pf the subscriptions sales
  order to view the the customer's perspective, where the customer can find the :guilabel:`Close
  Subscription` button on the left side of the sales order in the customer portal.
```

File: content/applications/sales/subscriptions/closing.rst [hunk 5]
```diff
@@ -1,105 +1,118 @@
...
-   :doc:`Create a quotation using subscription products <../subscriptions>`
-   to understand how subscriptions are managed in the **Odoo Subscriptions** application.
+  Customers can *only* chose a pre-configured reason why the subscription is being closed. They can
+  *not* enter a custom reason from the customer portal.
+
+Once the customer has chosen a close reason, they would click the :guilabel:`Submit` button on the
+pop-up window.
+
+When the subscription has been closed, a :guilabel:`Closed` tag is applied to the subscription order
+in the customer portal.
```
Comment (Felicious) [inline content/applications/sales/subscriptions/closing.rst:112]:
```suggestion
Upon closure, the subscription order in the customer portal is tagged as 
:guilabel:`Closed` tag.
```

**Review by Felicious — APPROVED** [review]
Amazing job with this doc, @ksc-odoo !

I only had a few minor rewording suggestions for brevity, so feel free to go through those and pick the ones you like (: 

The doc is already really clear and easy to follow, so after you go through the comments, feel free to pass this along to final review!

File: content/applications/sales/sales/send_quotations/quote_template.rst [hunk 6]
```diff
@@ -239,7 +239,7 @@ configurations into place.
 
 There is also a blue banner at the top of the quotation template design with a link to quickly
 return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation form in the
-back-end of the *Sales* application.
+back-end of the *Sales* application. placeement   compleeete.
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/quote_template.rst:242]: oh my gosh! that's when we were still testing out linter! haha guess it made its way onto the actual PR... oh boy!

**Comment by ksc-odoo** [comment]
@Felicious Thanks for all your awesome feedback! I made all your suggested edits, and believe this one is now ready for Tech Review from @samueljlieber , whenever you get a chance! Thanks! 👍

File: content/applications/sales/subscriptions/closing.rst [hunk 7]
```diff
@@ -1,105 +1,115 @@
...
+Configuration --> Recurring Plans`. From there, either create a new plan by clicking :guilabel:`New`
+or select an existing one to modify it.
+
+Once on the :guilabel:`Recurring Plans` form, enable the :guilabel:`Closable` option, in the
+:guilabel:`Self-Service` section, to allow customers to close their own subscriptions using the
+customer portal.
 
-.. image:: closing/configuration-to-close-your-subscriptions.png
+.. image:: closing/recurring-plans-closable-option.png
   :align: center
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/closing.rst:32]: The image attributes need to be lined up with the `i` in `.. image::`.
```suggestion
.. image:: closing/recurring-plans-closable-option.png
   :align: center
   :alt: The Closable option on a recurring plan form in Odoo Subscriptions.
```

File: content/applications/sales/subscriptions/closing.rst [hunk 8]
```diff
@@ -1,105 +1,115 @@
...
-  :alt: Close your subscription from a customer point of view with Odoo Subscriptions
+  :alt: The chatter of a churned sales order for a closed subscription in Odoo Subscriptions.
+
+Customer view
+-------------
+
+.. note::
+  Administrators can use the :guilabel:`Preview` button at the top pf the subscriptions sales order
+  to view the the customer's perspective, where the customer can find the :guilabel:`Close
+  Subscription` button on the left side of the sales order in the customer portal.
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/closing.rst:85]: The text in an admonition need to be lined up with the first character in the admonition type title, `n` in `.. note::`:
```suggestion
.. note::
   As an administrator, the ability to visualize what customers see when managing their
   subscriptions is accessible via the :guilabel:`Preview` button, located at the top of the
   subscription sales order.
```

File: content/applications/sales/subscriptions/closing/close-subscriptions-administrator.png
Comment (samueljlieber) [inline content/applications/sales/subscriptions/closing/close-subscriptions-administrator.png:1]: This image is too wide at `1111px`, please retake so it is at or below 933px wide.

File: content/applications/sales/subscriptions/closing/churned-sales-order.png
Comment (samueljlieber) [inline content/applications/sales/subscriptions/closing/churned-sales-order.png:1]: This image could be retaken with a little more width and zoom, it is currently `661px` wide and the text is pretty small.

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Nice job on this PR, the improvements in this doc look pretty great. I have just a few corrections to get this PR ready for final, please see below. I only listed a couple of the indentation corrections, so please look out for others throughout the doc. 

Let me know when this PR is ready for another look, thank you!

**Comment by ksc-odoo** [comment]
Alrighty, @samueljlieber , I made all your suggested edits and believe this one is ready for another look from you, whenever you get a chance. Thanks! 👍

File: content/applications/sales/subscriptions/closing.rst [hunk 9]
```diff
@@ -1,105 +1,115 @@
...
+are choosing to close the subscription.
 
-.. image:: closing/chatter-history-when-a-customer-closes-a-subscription.png
-  :align: center
-  :alt: What happens when customers close their subscription in Odoo Subscriptions?
+.. image:: closing/close-subscription-customer-pov.png
+   :align: center
+   :alt: The close subscription pop-up window customers see when closing a subscription.
 
 .. note::
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/closing.rst:102]: This note admonition's content needs to be indented by a space
```suggestion
.. note::
   Customers can *only* chose a pre-configured reason why the subscription is being closed. They can
   *not* enter a custom reason from the customer portal.
```

File: content/applications/sales/subscriptions/closing.rst [hunk 10]
```diff
@@ -1,105 +1,115 @@
...
+===================
+Close subscriptions
+===================
 
-Losing a customer is always difficult, especially if you put a lot of effort into getting them to
-sign up for your products/services. However, many companies come up with dubious methods to reduce
-the probability of this happening.
+While losing customers is not ideal, making it easy for customers to cancel subscriptions
+minimizes the impact of losing them, fostering a positive experience for the customer *and* the
+business.
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/closing.rst:7]: Early line break
```suggestion
While losing customers is not ideal, making it easy for customers to cancel subscriptions minimizes
the impact of losing them, fostering a positive experience for the customer *and* the business.
```

File: content/applications/sales/subscriptions/closing.rst [hunk 11]
```diff
@@ -1,105 +1,115 @@
...
+At that point, the ability to close the subscription becomes available, via the :guilabel:`Close`
+button at the top of the :guilabel:`In Progress` subscription order. This option is also available
+after the order has been invoiced and the payment has been registered.
 
-.. image:: closing/close-your-subscriptions-as-an-administrator.png
-  :align: center
-  :alt: Close your subscription from an administration point of view with Odoo Subscriptions
+.. image:: closing/close-subscriptions-administrator.png
+   :align: center
+   :alt: Close subscription from an administration point of view with Odoo Subscriptions.
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/closing.rst:53]: Line over 100 char
```suggestion
   :alt: The Close Reason pop-up that appears when the Close button is clicked in Odoo
         Subscriptions.
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo! Thank you for implementing my corrections, the PR looks good! I did catch a couple more line breaks and indentation corrections. Please make these before tagging for final. Approving now 👍

**Comment by ksc-odoo** [comment]
Thanks for the feedback @samueljlieber -- made all your suggested edits, and will tag this one for FINAL review now. Ready for ya whenever you get a chance, @StraubCreative 👍

File: content/applications/sales/subscriptions/closing.rst [hunk 12]
```diff
@@ -1,105 +1,114 @@
...
-   dissatisfied customers, and who retain their customers by making it easier for them to leave if
-   they want to. By giving them the opportunity to close their own subscriptions, your customers do
-   not feel trapped because they subscribed to your products/services.
-
-In summary, the **Odoo Subscriptions** application gives you the
-**possibility to choose what you want to apply**. Indeed, you can decide whether to give your customers the
-option to close their subscriptions whenever they want to or to restrict this possibility. It depends
-on you and we will show you how to do that in our amazing application.
+Odoo *Subscriptions* provides flexibility for businesses to decide whether customers can self-manage
+their subscriptions, or restrict that ability entirely.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/closing.rst:17]: What's here on lines 16-17, I'd argue, is the only part you need for the intro.
This sentence specifically (gh won't let me select):
```
Odoo *Subscriptions* provides flexibility for businesses to decide whether customers can self-manage
their subscriptions, or restrict that ability entirely.
```

The rest is borderline pre-sales expo or not needed.

File: content/applications/sales/subscriptions/closing.rst [hunk 13]
```diff
@@ -1,105 +1,114 @@
...
 
 Configuration
 =============
 
-Go to :menuselection:`Subscriptions --> Configuration --> Subscription templates`. From there, you can
-create a new *Subscription template* or modify an existing one. When editing your template, underneath
-the Invoicing tab, you have the possibility to activate the option *Closable by customer*.
-Once enabled, this option gives your customers the right to close their own subscriptions.
+To configure recurring plans offered, navigate to :menuselection:`Subscriptions app -->
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/closing.rst:22]: First part not needed or seems forced. Suggestion to just tell the reader to go to the right dash.
```suggestion
Start by navigating to :menuselection:`Subscriptions app -->
```

File: content/applications/sales/subscriptions/closing.rst [hunk 14]
```diff
@@ -1,105 +1,114 @@
...
+====================
 
 Administrator view
 ------------------
 
-Once confirmed, a quotation becomes a sales order and a new subscription is automatically created.
-Therefore, this subscription has the status *In progress*. From there, you have the possibility to
-close the subscription.
+After a quotation for a subscription product has been confirmed, it becomes a sales order, and the
+subscription is officially :guilabel:`In Progress`.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/closing.rst:44]: specificity
```suggestion
subscription status changes to :guilabel:`In Progress`.
```

File: content/applications/sales/subscriptions/closing.rst [hunk 15]
```diff
@@ -1,105 +1,114 @@
...
+
+At that point, the ability to close the subscription becomes available, via the :guilabel:`Close`
+button at the top of the :guilabel:`In Progress` subscription order. This option is also available
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/closing.rst:47]: "top of the **In Progress** subscription order" didn't make sense to me when I read it a couple times.
Here's a suggested rewrite: 

```suggestion
button at the top of the subscription order, near the row that contains :guilabel:`In Progress` and other stages. This option is also available
```

File: content/applications/sales/subscriptions/closing.rst [hunk 16]
```diff
@@ -1,105 +1,114 @@
...
+.. image:: closing/close-subscription-customer-pov.png
+   :align: center
+   :alt: The close subscription pop-up window customers see when closing a subscription.
 
 .. note::
-   Before closing a subscription, check out our documentation on how to
-   :doc:`Create a quotation using subscription products <../subscriptions>`
-   to understand how subscriptions are managed in the **Odoo Subscriptions** application.
+   Customers can *only* chose a pre-configured reason why the subscription is being closed. They can
+   *not* enter a custom reason from the customer portal.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/closing.rst:103]: Little more context.
```suggestion
   *not* enter a custom reason from the customer portal. These selections can be adjusted by
   navigating to :menuselection:`Subscriptions --> Configuration --> Close Reasons`.
```

File: content/applications/sales/subscriptions/closing.rst [hunk 17]
```diff
@@ -1,105 +1,114 @@
...
+
+Once the customer has chosen a close reason, they would click the :guilabel:`Submit` button on the
+pop-up window.
+
+Upon closure, the subscription order in the customer portal is tagged as :guilabel:`Closed` tag.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/closing.rst:108]: redundant
```suggestion
Upon closure, the subscription order in the customer portal is tagged as :guilabel:`Closed`.
```

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 
Approving with comments for your consideration.
Merge when ready, thanks! 👍 

@robodoo delegate=ksc-odoo

File: content/applications/sales/subscriptions/closing.rst [hunk 18]
```diff
@@ -1,105 +1,114 @@
...
 
-Losing a customer is always difficult, especially if you put a lot of effort into getting them to
-sign up for your products/services. However, many companies come up with dubious methods to reduce
-the probability of this happening.
+While losing customers is not ideal, making it easy for customers to cancel subscriptions minimizes
+the impact of losing them, fostering a positive experience for the customer *and* the business.
 
-.. raw:: html
+If the subscription closing process is made unnecessarily difficult or cumbersome, that will likely
+increase the level dissatisfaction amongst the leaving customer base, which can negatively impact
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/sales/subscriptions/closing.rst:9]: I'll just be keeping the lines of 16-17. Thanks for the advice. Removing the other lines in the intro.

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**